contents/docs/product-analytics/correlation.mdx

```diff
@@ -10,52 +10,40 @@ availability:
   enterprise: full
 ---
 
-Correlation analysis is a feature within [funnels](/docs/product-analytics/funnels) that **automatically** highlights significant factors impacting the conversion rate of users within a funnel.
+Correlation analysis is a feature within [funnels](/docs/product-analytics/funnels) that **automatically** highlights significant factors impacting conversion rate.
 
-Funnels have the ability to directly compare conversion rates of users for a specific property via the breakdown feature, but this requires you to know what property to investigate. If you're not certain, correlation analysis is helpful because it can highlight significant events or properties automatically. 
+Funnels have the ability to directly compare conversion rates of users for a specific property via the breakdown feature, but this requires you to know which property to investigate. If you're not certain, correlation analysis highlights significant events or properties for you. 
 
-Given a funnel, PostHog will calculate the conversion rate for different slices (either by events performed, properties they have or properties for the events they performed) and surface relevant signals. Below is an example of what correlation analysis might look like:
+As an example, the below insight shows that desktop users were `1.3x more likely` to convert than mobile or tablet users. Similarly, we also seee that users who completed the `uploaded_cv_for_review` event were `8.0x more likely` to convert.
 
-![Correlation analysis](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/docs/user-guides/funnels/funnel-correlation-analysis.png)
+<ProductScreenshot
+    imageLight = {"https://res.cloudinary.com/dmukukwp6/image/upload/v1716387676/posthog.com/contents/Screenshot_2024-05-22_at_3.20.17_PM.png"} 
+    imageDark = {"https://res.cloudinary.com/dmukukwp6/image/upload/v1716387676/posthog.com/contents/Screenshot_2024-05-22_at_3.20.33_PM.png"}
+    classes="rounded"
+    alt= "Example of a correlation analysis insight"
+/>
 
-## Using correlation analysis
+It's important to remember that the accuracy any correlation is dependent on the sample size. If you've only got a handful of users for a specific correlation, then it's likely that it will *not* be useful.
```

Comment:
```suggestion
It's important to remember that the accuracy of any correlation depends on the sample size. If you've only got a handful of users for a specific correlation, then it's likely that it will *not* be accurate.
```